src/site/components/up_to_top_button.html

```diff
@@ -13,15 +13,26 @@
 
 <div id="back-to-top-container">
   <div class="usa-grid usa-grid-full">
-    <div class="usa-width-three-fourths">
-      <div class="usa-content">
+    {% if isCampaignLandingPage %}
+      <div class="usa-content" style="max-width: 100rem;">
```

Comment:
Is this part of our usual style patterns? Is there a formally approved method of "exceptional" styling that we will need to follow to break pattern?

Reply:
I think traditionally we could use a custom class and make use of `shame.scss`, but I wasn't sure if that was better here tbh haha 😂

Reply:
instead of doing `style="max-width: 100rem;"` can we just use class name `usa-width-one-whole`? i'm not clear on what `100rem` is in this context though, just assuming it means full-width

Reply:
Unfortunately `usa-width-one-whole` does not work as a replacement for the `style="max-width: 100rem;"`:

![image](https://user-images.githubusercontent.com/12773166/113306429-4e018d00-92c1-11eb-858c-2202b7e88764.png)

![image](https://user-images.githubusercontent.com/12773166/113306293-2b6f7400-92c1-11eb-8da1-0007b886aa1b.png)

Reply:
I'd like to say I'm not happy about using the `style` attribute but I just literally could not figure it out without having custom css here (happy to make this custom class in `shame.scss` if we prefer) 😭 totally open to more suggestions though!

Reply:
i'm just wondering where `100rem` came from - it just looks right in how it lines up in CLP?

Reply:
> it just looks right in how it lines up in CLP?
Yupp! 💯 

> i'm just wondering where 100rem came from
From me messing with the css in dev tools far too long yesterday 😂

Reply:
This has to align with a grid class though, right? Like since the whole CLP page is a series of grids, it seems like we just need to write the grid (instead of 100rem) to make the button align.

Sorry, just having a hard time following this one.